CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 Fix: Static attributes from service not applied if device has static attributes (#757)
 Move Docker secret support inside the Node Application - remove Entrypoint (#885)
 Fix: IOTA_EXPLICIT_ATTRS env var was not working
+Add lax validation mode for OPC-UA IoT Agent using IOTA_RELAX_TEMPLATE_VALIDATION (#920)
```

Comment:
But the new parameter could be used by any iotagent, not just opc-ua iotagent, could be?

Reply:
From a language point of view of course you are right ;) However, it's a matter of style. We try to describe the features in the changelog of the library in a IOTA-neutral way.

So, suggestion:

```suggestion
Add lax validation mode using IOTA_RELAX_TEMPLATE_VALIDATION (#920)
```